### ISSUE TITLE
Add Qualified Opportunity Zones

### ISSUE THREAD

**Issue by stdavis** [agrc/locate#166]
- [ ] Place within the "Economy" section right after "Enterprise Zones".
- [ ] Label similar to the GOED website, "[OZ], [CityName]".
- [ ] Add to summary report including POC similar to the Tax Increment Zone and link to both [GOED's opportunity zones pages](https://business.utah.gov/opportunity-awaits-statewide-opportunity-zones-announced/) and the [Treasury Department's page](https://home.treasury.gov/news/press-releases/sm0414).

@nathankota: Did I collect these requirements accurately? Am I missing anything?

**Comment by nathankota** [comment]
That looks complete and correct Scott. Thank you.

**Comment by stdavis** [comment]
What's the name of the field containing the ID that is to show up on the label? Is it the tract ID (GEOID)?
![image](https://user-images.githubusercontent.com/1326248/43612159-b105bf78-9668-11e8-9b7d-91a353b687c1.png)

**Comment by nathankota** [comment]
Yes, looks  like it's the 11 digit GEOID. They also want the associated city, comma separated from the GEOID, like it appears on https://business.utah.gov/opportunity-awaits-statewide-opportunity-zones-announced/. I can't tell from the table screen grab above if the associated city is in there or not.

**Comment by stdavis** [comment]
I'm guessing that the city is `ZoneName`:
![image](https://user-images.githubusercontent.com/1326248/43647135-fccbc8b8-96f3-11e8-9431-b01fd640b6c0.png)

**Comment by stdavis** [comment]
So labels will be: `[GEOID], [ZoneName]`.
